applications/DEMApplication/test_examples/DEM_run_all_benchmarks_analysis.py

```diff
@@ -84,10 +84,7 @@ def Run():
                 if platform.system()=="Windows":
                     subprocess.check_call(["python", path + "/DEM_benchmarks_analysis.py", str(benchmark), ">", "BenchTemp.info"], stdout=f, stderr=f)
                 else:
-                    if sys.version_info >= (3, 0):
-                        subprocess.check_call(["python3", path + "/DEM_benchmarks_analysis.py", str(benchmark), ">", "BenchTemp.info"], stdout=f, stderr=f)
-                    else:
-                        subprocess.check_call(["python", "-3", path + "/DEM_benchmarks_analysis.py", str(benchmark), ">", "BenchTemp.info"], stdout=f, stderr=f)
+                    subprocess.check_call(["python3", path + "/DEM_benchmarks_analysis.py", str(benchmark), ">", "BenchTemp.info"], stdout=f, stderr=f)
```

Comment:
Note: On some systems the command to launch `python3` is `python` (e.g. win in the CI)
I came up with [this solution](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CoSimulationApplication/tests/test_co_sim_io_py_exposure.py#L294) which seems to work nicely
(FYI @roigcarlo I think this could also be interesting for you)

Reply:
Good point. I added your check in the DEM. I did not dare to make that modification in the 7 other places where this happens, though.